svelte-kit-scss/README.md

```diff
@@ -35,19 +37,19 @@ This starter kit features **Svelte** and **SCSS**
 
 ### Architecture Decisions
 
-For this kit, we maintained the [Sveltekit Project Strucuture](https://kit.svelte.dev/docs/project-structure) and we do recommend it. For the `src/lib/components` folder, each component is co-located with its tests and stories. This structure makes it easy to find all the code and functionality related to a specific component. Due to Sveltekit's preference for data fetching from the route page, the components are primarily simple view components.
+For this kit, we maintained the [Sveltekit Project Structure](https://kit.svelte.dev/docs/project-structure) and we do recommend it. For the `src/lib/components` folder, each component is co-located with its tests and stories. This structure makes it easy to find all the code and functionality related to a specific component. Due to Sveltekit's preference for data fetching from the route page, the components are primarily simple view components.
 
 - #### Storybook
 
   Storybook was introduced because it is really a great tool for testing components in isolation and good for documentation.
 
 - #### Styling
 
-  Svelte comes with sass support. Sass being one of the most mature, stable, and powerful professional grade CSS extension language, it quickly allows us to design robust components.
+  Svelte comes with sass support. Sass one of the most mature, stable, and powerful professional-grade CSS extension language, it quickly allows us to design robust components.
```

Comment:
i think the original sounds better :thinking:

Reply:
Svelte comes with sass support. Sass is one of the most mature, stable, and powerful professional-grade CSS extension languages, it quickly allows us to design robust components.